definitions/windows.yaml

```diff
@@ -1030,6 +1030,19 @@ sources:
 supported_os: [Windows]
 urls: ['https://technet.microsoft.com/en-us/library/cc939701.aspx']
 ---
+name: WindowsPolicyDisallowRun
+doc: |
+  Restrict users from running specific applications.
+  Typically used by malware to block AV application.
```

Comment:
"Typically used by malware to block AV." and I think it will fit on one line then.